vignettes/ConvertingPharmacoSetToGDR.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "Converting PharmacoSet Drug Response Data into gDR object" 
+author:
+- name: Jermiah Joseph
+  email: jermiah.joseph@uhn.ca
+- name: Bartosz Czech
+  email: bartosz.w.czech@gmail.com
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{ConvertingPharmacoSetDrugResponseDataintogDRobject}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(PharmacoGx)
+library(gDR)
+library(gDRcore)
+library(gDRimport)
+```
+
+
+# Overview
+The `gDRimport` package is a part of the gDR suite. It helps to prepare raw drug response data for downstream processing. 
+It mainly contains helper functions for importing/loading/validating dose response data provided in different file formats. 
+In collaboration with the BHKLab, `gDRimport` also provides functions that can convert a `PharmacoGx::PharamcoSet` object into a gDR object. 
+With this functionality, users familiar with the gDR suite of packages and methods can utilize the publically available, curated datasets from the PharmacoGx database. 
+The main step in this process is to extract the drug dose-response data from the PharmacoSets and transform them into a `data.frame` that can be used as input for the `gDRcore::runDrugResponseProcessingPipeline`.
+
+
+# Loading a PharmacoSet (PSet)
+Whereas a user might already have a pharmacoset loaded in their R session, if they wish to obtain a different pharmacoset or use the same script in the future we provide a helper function to do so. 
+It helps to have a user directory in which to store all pharmacosets, and by passing this directory into the function as a parameter, the function will also check to see if the PSet exists in the user-definded directory.
+This is to ensure that the PSet is not being re-downloaded if it already has.  
+```{r eval = FALSE}
+pset <- gDRimport:::.getPSet("Tavor_2020")
```

Comment:
Maybe we should export the function?

Reply:
You can simplify the whole chunk with the solution @darsoo worked out for the examples: https://github.com/gdrplatform/gDRimport/blob/main/R/pset_to_gdrDF.R#L11-L14.

Reply:
Done